staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -959,6 +958,32 @@ func (r *crdHandler) getOrCreateServingInfoFor(uid types.UID, name string) (*crd
 	return ret, nil
 }
 
+func scopeWithFieldManager(typeConverter fieldmanager.TypeConverter, reqScope handlers.RequestScope, storage rest.ResetFieldsStrategy, ignoreManagedFieldsFromRequestObject bool) (handlers.RequestScope, error) {
+	resetFields := map[fieldpath.APIVersion]*fieldpath.Set{}
+	// storage is an interface, in the event it holds a nil value
+	// storage itself will not equal nil.
+	// We must check that the value if storage is not nil in order to avoid a panic.
```

Comment:
This is almost certainly due to an earlier mistake. It's very rude of us to store nil pointers like this, can you find and fix the culprit instead of papering over it here?